src/bench/crypto_hash.cpp

```diff
@@ -199,6 +200,98 @@ static void SipHash_32b(benchmark::Bench& bench)
     });
 }
 
+static void SaltedOutpointHasherBenchmark_hash(benchmark::Bench& bench)
+{
+    FastRandomContext rng(true);
+    constexpr size_t size = 1000;
```

Comment:
In "bench: Add COutPoint and SaltedOutpointHasher benchmarks" 6ce13a33c6901d820467b9c24a03ca3861d91640

You could factor out `size` as a static global variable and reuse it in the benchmarks to avoid repetition!

Reply:
I thought of that but they each have different difficulties, so they should be tuned separately (i.e. if something takes 2x as much time per round, we likely want fewer repetitions)